config/service_providers.yml.old

```diff
@@ -4,7 +4,8 @@ test:
     assertion_consumer_logout_service_url: 'http://localhost:3000/test/saml/decode_slo_request'
     sp_initiated_login_url: 'http://localhost:3000/test/saml'
     block_encryption: 'none'
-    cert: 'saml_test_sp'
+    certs:
```

Comment:
**Question:** What is the purpose of this file?

Reply:
🤷 no idea, maybe an archive from before we moved to `identity-idp-config`?

Reply:
I was going to ask if we should just take it out but that seems out of scope for this issue 😛.

Reply:
meh seems easy enough to remove, might as well